Formula/dafny.rb

```diff
@@ -20,13 +20,22 @@ class Dafny < Formula
   depends_on "mono-libgdiplus" => :build
   depends_on "nuget" => :build
   depends_on "mono"
-  depends_on "z3"
+  # Use the following when dafny uses the latest z3 (and omit the 
```

Comment:
```suggestion
  # Use the following when dafny uses the latest z3 (and omit the
```
CI is failing because of the trailing whitespace here.

Reply:
Thanks. Fixed.